introduction/intro.rst

```diff
@@ -154,31 +142,15 @@ software, because the fundamental operating system that constitutes the
 core infrastructure of our digital lives **must** be free and
 open-source in order to be trustworthy.
 
+.. image:: /attachment/doc/r4.0-snapshot12.png
+   :alt:
 
-.. figure:: /attachment/doc/r4.0-snapshot12.png
-   :alt: Qubes desktop screenshot
-
-
-
-Video Tours
-~~~~~~~~~~~
-
-Want to see Qubes OS in action? Sit back and watch a guided :doc:`tour! </introduction/video-tours/>`
-
-
-Screenshots
-~~~~~~~~~~~
-
-See what using Qubes actually looks like with these :doc:`screenshots </introduction/screenshots/>` of various
-applications running in Qubes.
-
-
-Getting Started
-~~~~~~~~~~~~~~~
-
-Ready to get started with Qubes? :doc:`Here's </introduction/getting-started>` what you need to know after installing.
-
+Qubes OS at a glance
+--------------------
```

Comment:
I think this suggestion of reordering is nice.
But would suggest waiting for after the QA time frime, since it is a contents' change